executor_manager/executors/docker/executor.py

```
@@ -371,6 +371,22 @@ def _create_new_container(
                 self._check_container_health(task, executor_name, is_validation_task)
 
+            # For non-sandbox tasks, dispatch the first HTTP request after startup.
+            # Sandbox containers are intentionally started idle and wait for /execute requests.
+            if not is_sandbox_task:
+                try:
+                    self._dispatch_initial_task_to_new_container(task, executor_name)
+                except Exception:
+                    # Avoid leaking an idle container when initial dispatch fails.
+                    try:
+                        delete_container(executor_name)
+                    except Exception as cleanup_error:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)